src/pr.ts

```diff
@@ -29,8 +30,11 @@ export const raisePR = async (forkBranchName: string, targetBranch: string, extr
     title: `chore: bump libcc (${targetBranch})`,
     body: `Updating libcc reference to latest.  Changes since the last roll:
 
-${extraCommits.map((commit) =>
-      `* [\`${commit.sha.substr(0, 8)}\`](${COMMIT_URL_BASE}/${commit.sha}) ${commit.message}`).join('\n')}
+${extraCommits.map((commit) => {
+      const sha = `[\`${commit.sha.substr(0, 8)}\`](${COMMIT_URL_BASE}${commit.sha})`;
+      const msg = commit.message.replace(/#(\d+)/g, `${ISSUE_URL_BASE}$1`);
```

Comment:
This regex should match all digits followed by a space to avoid some mis-matches

Reply:
I've improved the regex to avoid mis-matches. I list some valid examples.
- `brahbrah #XXX`
- `brahbrah [#XXX]`
- `brahbrah(#XXX)`
- `#XXX brahbrah`